lib/helpers.bash

```diff
@@ -77,7 +77,8 @@ bash-it ()
         _bash-it-search $component "$@"
         return;;
       update)
-        func=_bash-it_update;;
+        _bash-it_update $component "$@"
```

Comment:
Why did you add the `$component` here? Is this really needed for the update scenario?

Reply:
removed in the currrent version